qa/src/main/resources/org/glassfish/qa/config/checkstyle/checkstyle.xml

```diff
@@ -82,6 +82,7 @@
         <!-- Checks for imports                              -->
         <!-- See http://checkstyle.sf.net/config_import.html -->
         <module name="IllegalImport" /> <!-- defaults to sun.* packages -->
+        <module name="RedundantImport" />
```

Comment:
I agree, if it is fixed everywhere, but not by using package names like java.lang.Object in code. If there are conflicts, ie. org.glassfish.Object, I would prefer JDK in imports and conflicting class with full name in code (except if there would be 100:1 occurences).

Reply:
Conflicts resolved by replacing on-demand imports with individual ones.